src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
```

Comment:
```suggestion
```
you do not need this `div`, you can just add class `stars` to the stars--0, stars--1, ... , stars--5